CHANGELOG.md

```diff
@@ -3,13 +3,14 @@
 For official release notes for Docker Engine CE and Docker Engine EE, visit the
 [release notes page](https://docs.docker.com/engine/release-notes/).
 
-## 19.03.6 (2020-01-23)
+## 19.03.6 (2020-02-05)
 
 ### Builder
 
 - builder-next: Allow modern sign hashes for ssh forwarding. [docker/engine#453](https://github.com/docker/engine/pull/453)
 - builder-next: Clear onbuild rules after triggering. [docker/engine#453](https://github.com/docker/engine/pull/453)
-- bump hcsshim to fix docker build failing on Windows 1903. [docker/engine#429](https://github.com/docker/engine/pull/429)
+- Bump hcsshim to fix docker build failing on Windows 1903. [docker/engine#429](https://github.com/docker/engine/pull/429)
+- Fix issue with directory permissions when BuildKit and usernamespaces are enabled. [moby/moby#40440](https://github.com/moby/moby/pull/40440)
```

Comment:
@arkodg sorry didn't see the `builder-next:` prefix. This one should have it as well, and we can thus `s/BuildKit and //` and `s/are enabled/is enabled/`